### PR TITLE
feat: implement Header and Hero section

### PR DESCRIPTION
**Pull request by egokiemute**
I just created the Header and Hero sections in this pull request

### CODE REVIEW

File: src/layouts/Layout.astro [hunk 1]
```diff
@@ -13,39 +14,38 @@ const { title } = Astro.props;
 		<meta name="description" content="Astro description" />
 		<meta name="viewport" content="width=device-width" />
 		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+		<link href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@300..700&display=swap" rel="stylesheet">
```
Comment (codejagaban) [inline src/layouts/Layout.astro]: Font should also include `inter` for body fonts

File: src/sections/PrevImage.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
+
```
Comment (codejagaban) [inline src/sections/PrevImage.tsx]: This can be an Astro component, it uses very minimal JS

File: src/sections/PrevImage.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+export default function PrevImage() {
+  return (
+    <div className="flex flex-col gap-2 w-full bg-bg-primary overflow-hidden">
+      <div
+        className="flex gap-1 w-full"
+      >
+        {firstColumn.map(({ imageSrc, altText }) => (
+          <div key={altText} className="min-w-[25%]">
+            <img
```
Comment (codejagaban) [inline src/sections/PrevImage.tsx]: Use Astro image component to serve Optimized images

File: src/sections/PrevImage.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            />
+          </div>
+        ))}
+      </div>
+      <div
+        className="flex gap-1 w-full"
+      >
+        {secondColumn.map(({ imageSrc, altText }) => (
+          <div key={altText} className="min-w-[20%]">
+            <img
```
Comment (codejagaban) [inline src/sections/PrevImage.tsx]: Same with the images here

File: src/sections/Header.tsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
```
Comment (codejagaban) [inline src/sections/Header.tsx]: This component should be in the `layouts` directory

File: src/sections/PrevImage.tsx [hunk 6]
```diff
@@ -0,0 +1,76 @@
+
+
```
Comment (codejagaban) [inline src/sections/PrevImage.tsx]: Ideally, this component should be called `PrevEventImages`

**Review by codejagaban — CHANGES_REQUESTED** [review]
This is a good start, just some few changes.

nice job

File: src/layouts/Layout.astro [hunk 7]
```diff
@@ -13,39 +14,38 @@ const { title } = Astro.props;
...
-		--accent-gradient: linear-gradient(
-			45deg,
-			rgb(var(--accent)),
-			rgb(var(--accent-light)) 30%,
-			white 60%
-		);
+		--bg-primary: #121121;
+		--bg-light: #CCCCCC;
+		--bg-secondary: #F8F8FF;
+		--bg-azure: #018FFB;
```
Comment (codejagaban) [inline src/layouts/Layout.astro]: are we really using these values, if "No", we should remove them

Comment (egokiemute) [reply src/layouts/Layout.astro]: Will use them later in the other sectons

File: src/sections/PrevImage.tsx (hunk 6 above)
Comment (egokiemute) [reply src/sections/PrevImage.tsx]: Renamed

File: src/sections/Header.tsx (hunk 5 above)
Comment (egokiemute) [reply src/sections/Header.tsx]: Moved to the Layouts

File: src/layouts/Layout.astro (hunk 1 above)
Comment (egokiemute) [reply src/layouts/Layout.astro]: Added

**Review by codejagaban — APPROVED** [review]
LGTM

**Pull request merged**